src/components/ListingDetails/index.tsx

```diff
@@ -0,0 +1,320 @@
+          <DateText $bold>
+            {dateHeader}: <DateText> {dateData}</DateText>
+          </DateText>
+        </IconTextGroup>
+      );
+    } 
+  };
+
+  const profile = useContext(ProfileContext);
+
+  const Interest = useMemo(() => {
+    if (profile && profile.userId)
+      return profile.profileData ? (
+        // <InterestForm caseData={listingData} />
+        <H3>Updated Interest Form Coming Soon!</H3> // remove after refactoring InterestForm
+      ) : (
+        <>
+          <H3>
+            Please finish submitting your profile before submitting interest.
+          </H3>
+          <AuthButtons>
+            <LinkButton
+              $primaryColor={COLORS.blueMid}
+              $secondaryColor={COLORS.blueDark}
+              href="/onboarding/roles"
+            >
+              Go to Onboarding
+            </LinkButton>
+          </AuthButtons>
+        </>
+      );
+
+    return (
+      <>
+        <H3>
+          You aren&apos;t signed in. In order to submit your interest, log into
+          your account, or create a new account.
+        </H3>
+        <AuthButtons>
+          <LinkButton $secondaryColor={COLORS.blueMid} href="/signup">
+            Sign Up
+          </LinkButton>
+
+          <LinkButton
+            $primaryColor={COLORS.blueMid}
+            $secondaryColor={COLORS.blueDark}
+            href="/login"
+          >
+            Log In
+          </LinkButton>
+        </AuthButtons>
+      </>
+    );
+  }, [listingData, profile]);
+
+  return (
+    <CaseDisplay>
+      <InfoContainer>
+        <BorderedSection>
+          {dateComponent()}
+          <Flex $align="center" $gap="18px" $direction="row">
+            <img src={logo.src} alt="DOHS Logo" width="77" height="77" />
+            <H2>{listingData.title || 'Migrant seeking representation'}</H2>
+          </Flex>
+        </BorderedSection>
+        <BorderedSection>
+          <IconTextGroup>
+            <Icon type="tag" />
+            <H4>
+              {listingData.listing_type === 'LCA' ? 'Assignment' : 'Case'}{' '}
+              Highlights
+            </H4>
+          </IconTextGroup>
+          <InnerFieldContainer>
+            {listingFields().map(({ label, getValue }) => (
+              <div key={label}>{renderField(label, getValue(listingData))}</div>
+            ))}
+          </InnerFieldContainer>
+        </BorderedSection>
+        {listingData.listing_type === 'LCA' && (
+          <BorderedSection>
+            <IconTextGroup>
+              <Icon type="tag" />
+              <H4>Research Topic(s)</H4>
+            </IconTextGroup>
+            <P>
+              {listingData.research_topic
+                ? listingData.research_topic
+                : 'Not Available'}
+            </P>
+          </BorderedSection>
+        )}
+        <BorderedSection>
+          <IconTextGroup>
+            <Icon type="briefcase" />
+            <H4>
+              {listingData.listing_type === 'LCA'
```

Comment:
Noticing that One-Time Transalations & Document Translations have case-specific labels ("Case Description" and "Case Highlights"), and I think this could get confusing for the user. Could we update this label as well as the Highlights label so that:
-  for One-Time Interpretations the labels are "Interpretation Description" and "Interpretation Highlights"
- for Document Translations the labels are "Assignment Description" and "Assignment Highlights"

Reply:
just to confirm, after last worksession, have we confirmed that we will standardize the naming across all listing types as "Description" and "Highlights" rather than "Case Highlights" or "Assignment Highlights"?

Reply:
Confirmed: "Highlights" and "Description"